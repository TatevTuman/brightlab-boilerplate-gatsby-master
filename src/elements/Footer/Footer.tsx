import React, { memo } from 'react'
import { Container } from '@elements'
import { useSiteMetadata } from '@hooks'

export interface FooterProps {}

const Footer: React.FC<FooterProps> = props => {
  const { navigation } = useSiteMetadata()

  return (
    <div className={'footer'} role="contentinfo">
      <Container>
        <h2>Footer</h2>
      </Container>
    </div>
  )
}

export default memo(Footer)
