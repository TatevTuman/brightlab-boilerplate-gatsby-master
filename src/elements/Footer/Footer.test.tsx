import React from 'react'
import { render, cleanup, waitFor } from '@testing-library/react'
import Footer from './Footer'

// beforeAll(() => {})
// afterAll(() => {})
// beforeEach(() => {})
afterEach(() => {
  cleanup()
  jest.clearAllMocks()
})

describe('Footer', () => {
  it('renders correctly', async () => {
    const { container } = render(<Footer />)
    await waitFor(() => container)

    expect(container).toMatchSnapshot()
  })
})
