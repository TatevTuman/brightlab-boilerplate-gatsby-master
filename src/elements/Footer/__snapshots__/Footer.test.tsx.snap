// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <div
    class="footer"
    role="contentinfo"
  >
    <div
      class="container"
    >
      <nav
        class="navigation"
        role="navigation"
      >
        <ul
          class="navigation-list"
        >
          <li
            class="navigation-list-item"
            data-active="true"
          >
            <div
              class="navigation-list-item-icon"
            />
            <div
              class="navigation-list-item-link"
            >
              Home
            </div>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</div>
`;
